api/CaptureController.json

```diff
@@ -0,0 +1,119 @@
+{
+  "api": {
+    "CaptureController": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/CaptureController",
+        "spec_url": "https://w3c.github.io/mediacapture-screen-share/#dom-capturecontroller",
+        "support": {
+          "chrome": {
+            "version_added": "109"
+          },
+          "chrome_android": "mirror",
```

Comment:
`CaptureController` is NOT available on mobile, only on desktop as getDisplayMedia is supported only on Chrome for desktop. See https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/platform/runtime_enabled_features.json5;l=468;drc=1bdc259c97e0cfc63a958ec6eacf49640342777c

Reply:
Fixed